specification/README.md

```diff
@@ -0,0 +1,65 @@
+# Docker App Specification
+
+This section describes all the requirements for docker-app interoperability.
+
+## YAML Documents
+
+A Docker App is a set of 3 YAML documents:
+* `metadata`
+* `docker-compose`
+* `settings`
+
+These documents can be split in 3 different files or merged into one YAML file, using the [multi document YAML feature](http://yaml.org/spec/1.2/spec.html#id2760395).
+
+### metadata.yml
+
+`metadata.yml` defines some informations to describe the application in a standard `YAML` file.
+See [JSON Schemas](schemas/) for validation.
+
+### docker-compose.yml
+
+`docker-compose.yml` is a standard [Compose file](https://docs.docker.com/compose/compose-file/) with variable replacement.
+`Compose` minimal version is **v3.2**, see [JSON Schemas](https://github.com/docker/cli/tree/master/cli/compose/schema/data) for validation.
+
+### settings.yml
+
+`settings.yml` is a simple Key-Value file used to replace the variables defined in the `docker-compose` file. As it is an opened document, there is no schema for this one.
+
+## Validation
+
+The tool `yamlschema` included in `cmd/yamlschema` helps you validate a `YAML document` against its `JSON Schema`. 
```

Comment:
I think the specification should describe the details, rather than any tooling which might use them. I'd probably split this off into an appendix. ie. if someone was implementing the specification this isn't required.

Reply:
You mean another file? Like Appendix.md next to this README.md?